app/assets/stylesheets/modules/top_nav.scss

```diff
@@ -2,18 +2,30 @@
 @import "mixins";
 
 .top-bar {
-  h1 a {
+  height: 3.2rem;
+  background: transparent;
+
+  .name li h1 a {
     font-family: $dogstay-font-theme;
     font-size: 1.5em;
+    font-weight: bold;
+    color: #000000;
```

Comment:
Color literals like `#000000` should only be used in variable declarations; they should be referred to via variable everywhere else.